### PR TITLE
Cleanup docs and add new page for checks.

### PR DESCRIPTION
**Pull request by dlorenc**

### CODE REVIEW

File: checks.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+# Check Documentation
+
+This page contains information on how each check works and calculates scores.
+All of these checks are basically "best-geusses" currently, and operate on a series of heuristics.
```
Comment (inferno-chromium) [inline checks.md:4]: typo: guesses
s/series/set

**Review by kimsterv — APPROVED** [review]
Lgtm

File: checks.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+This check tries to determine if a project has declared and pinned its dependencies.
+It works by looking for a set of well-known package management lock files.
+
+## Signed-Tags
+
+This check looks for cryptographically signed tags in the git history.
+
+## Signed-Releases
+
+This check tries to determine if a project cryptographically signs release artifacts.
```
Comment (inferno-chromium) [inline checks.md:30]: s/signs/signed

File: checks.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+## Fuzzing
+
+This check tries to determine if the project uses a fuzzing system.
+It currently works by checking if the repo name is in the [OSS-Fuzz](https://github.com/google/oss-fuzz) project list.
+
+## SAST
+
+This check tries to determine if the project uses static analysis systems.
+It currently works by looking for well-known results (CodeQL, etc.) in GitHub pull requests.
```
Comment (inferno-chromium) [inline checks.md:61]: nit:probably linkify CodeQL

**Review by inferno-chromium — APPROVED** [review]

**Pull request merged**